commands_cancelSetBuy.go

```diff
@@ -55,5 +56,30 @@ func (csb cancelSetBuyCmd) ToAuditEvent() types.AuditEvent {
 }
 
 func (csb cancelSetBuyCmd) Execute() {
-	consoleLog.Warning("Not implemented: CANCEL_SET_BUY")
+	autoTxKey := types.AutoTxKey{
+		Stock:  csb.stock,
+		UserID: csb.userID,
+		Action: "Sell",
+	}
+	delete(workATXStore, autoTxKey)
+	ch, err := rmqConn.Channel()
+	failOnError(err, "Failed to open a channel")
+	defer ch.Close()
```

Comment:
Creating a new channel every time one of the trigger commands executes results in ~10x slowdown on prod.

One way to avoid this would be to create a goroutine whose only job is to push messages to the `autoTx` queue. It would be similar to the method for [publishing logs](https://github.com/DistributedDesigns/worker/blob/master/txWorker.go#L81).